### ISSUE TITLE
Fatal Error - scrape engine launch failed. thank you to check your antivirus

### ISSUE THREAD

**Issue by TMNTturtleguy** [Universal-Rom-Tools/Universal-XML-Scraper#178]
I have been using this program with great results for 2 days, and now i am getting this error every time i try to start a scrape.  Any ideas?

**Comment by OrlandoPete** [comment]
I've been getting this error a lot last two days as well, but earlier today it was fine. Now it is acting up again.

**Comment by Universal-Rom-Tools** [comment]
As I sayed, UXS was developped in "AUTOIT" a small language used sometime to make "malware"...
So Antivirus systematically put Autoit Dev in potential malware.

I already contacted a lot of antivirus compagny to say it's a false positive.
Some already respond and it's ok, some don't.

I can see it's not every time, but some time Antivirus block UXS engine (or the main UXS exe) but it's not every time (son't ask why ^^ I really don't know).

The easiest way is to put UXS and scraper.exe (or scraper64.exe) in the AV ignore list...
